tests/system/presenter/test_get_user_scope.py

```diff
@@ -84,9 +84,7 @@ def test_good(self) -> None:
     def test_without_user_None(self) -> None:
         status_code, data = self.request("get_user_scope", {"user_ids": [None]})
         self.assertEqual(status_code, 400)
-        self.assertIn(
-            "There is no user_id given to get the user_scope!", data["message"]
-        )
+        self.assertIn("data.user_ids[0] must be integer", data["message"])
```

Comment:
I would prefer the old message. It is a lot easier to understand and find for anyone

Reply:
Yes, but this can be said for all the fastjsonschema messages and we use them everywhere. Implementing additional code for a custom error message instead of just using the schema like everywhere else should IMO only be done in very special cases where it's expected that the error message will often be displayed to the user, which is not the case here.